### PR TITLE
Restricted settings help page.

### PR DESCRIPTION
**Pull request by BlockArchitech**
Adds the restricted settings help page. Hooray!

### CODE REVIEW

File: _topic/android-restricted-settings.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Restricted settings on Android."
+prompt: "Learn how to update restricted settings on Android for the Pebble app to work properly on Android 13+."
+keywords: "android app settings notifications"
+date:   2023-01-03 06:53:00
+author: "BlockArchitech"
+hideFirstStepNumber: false
+osSpecific: true
```
Comment (Willow-Systems) [inline _topic/android-restricted-settings.md:9]: This should be 'false'. 
OSSpecific means that there is a separate Android/iOS guide and will prompt the user for their OS (like /setup does). As there's _only_ a guide for Android, please leave as false

File: _topic/android-restricted-settings.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+os: android
+---
+
+# 1. Restricted Settings
+On Android 13 and higher, restricted settings prevents the Pebble app from working properly.
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/restricted.png)
```
Comment (Willow-Systems) [inline _topic/android-restricted-settings.md:17]: Please put the images in the /images folder

File: _topic/android-restricted-settings.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+# 2. Open Settings
+
+Open the settings app on your device.
```
Comment (Willow-Systems) [inline _topic/android-restricted-settings.md:21]: I think this step could be merged with step 3.

"Open settings, find the apps menu and tap all apps"

It should also point out that on some Android distros things might look slightly different

File: _topic/android-restricted-settings.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Search for the pebble app in the apps list.
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/search_pebble.png)
+
+# 5. Open the hamburger menu
+
+Open the hamburger menu, and tap "Enable Restricted Settings". You'll have to enter your screen lock for this.
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/hamburger.png)
```
Comment (Willow-Systems) [inline _topic/android-restricted-settings.md:44]: Could we put a circle or something around the hamburger menu to make it super obvious?
(also pls move images to /images)

**Review by Willow-Systems — CHANGES_REQUESTED** [review]
Thank you so much for adding this! 
I've highlighted a few changes that need to be made, once that's done I'll merge :)

File: _topic/android-restricted-settings.md (hunk 1 above)
Comment (Willow-Systems) [reply _topic/android-restricted-settings.md:9]: I'll update the help.rebble.io/contributing page to make it clearer that this should only be true when there's an iOS AND an Android set of instructions.

File: _topic/android-restricted-settings.md (hunk 4 above)
Comment (BlockArchitech) [reply _topic/android-restricted-settings.md:44]: Didn't see /images, I'll get those moved over.

File: _topic/android-restricted-settings.md (hunk 3 above)
Comment (BlockArchitech) [reply _topic/android-restricted-settings.md:21]: Will do!

**Comment by BlockArchitech** [comment]
Okay @Willow-Systems - that should be everything!

**Review by Willow-Systems — APPROVED** [review]
Looks good to me

**Comment by Willow-Systems** [comment]
Okay, it's live but not quite right. Could you please change the following:

- Change the prompt to 'Disable restricted settings on Android 13+'
- Remove the step numbers, as those are auto generated

**Comment by BlockArchitech** [comment]
new pull incoming!

**Pull request merged**